src/Helper/Settings.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Drupal\os2forms_fasit\Helper;
+
+use Drupal\Core\KeyValueStore\KeyValueFactoryInterface;
+use Drupal\Core\KeyValueStore\KeyValueStoreInterface;
+use Drupal\os2forms_fasit\Exception\InvalidSettingException;
+use Drupal\os2forms_fasit\Form\SettingsForm;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * General settings for os2forms_fasit.
+ */
+final class Settings {
+  /**
+   * The store.
+   *
+   * @var \Drupal\Core\KeyValueStore\KeyValueStoreInterface
+   */
+  private KeyValueStoreInterface $store;
+
+  /**
+   * The key value collection name.
+   *
+   * @var string
+   */
+  private $collection = 'os2forms_fasit.';
```

Comment:
The collection name should not end with a `.`. This is a bug you should not inherit.